### PR TITLE
Create Index page

### LINKED ISSUE #37: Index page
Create index page and needed components, based on the graphic proposal.

### PR DESCRIPTION
**Pull request by mriios**
## Issue
Fixes #37 

## Description
Create index/home page for website, based on graphic proposal

## Testing instructions
`yarn dev` then go to `http://localhost:3000/`

## Screenshots (optional)
image

### CODE REVIEW

**Comment by mriios** [comment]
sorry for no commits on this one ): forgot to make them every so often

**Comment by mriios** [comment]
Last commit fixes #39

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
 		"build": "next build",
 		"start": "next start",
 		"lint": "next lint",
-		"sb": "start-storybook -p 6006",
+		"sb": "start-sotorybook -p 6006",
```
Comment (ficolo) [inline package.json:10]: typo

**Review by ficolo — CHANGES_REQUESTED** [review]
Some small changes in tow files requested. Minor code style things.

File: src/components/FeedbackSection/FeedbackSection.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+import Form from "../Form/Form";
+import Input from "../Input/Input";
+import InputAndLabel from "../Input/InputAndLabel";
+
+interface IFeedbackSectionProps {
+	backgroundColor: "light" | "dark" | "white";
+}
+
+const FeedbackSection = (props: IFeedbackSectionProps) => {
+	let backgroundColorClass;
```
[the remark this answers is not in the document]
Comment (mriios) [reply src/components/FeedbackSection/FeedbackSection.tsx:12]: noted! Will take it into consideration for next time

**Review by ficolo — APPROVED** [review]

**Comment by ficolo** [comment]
Looks great!

**Pull request merged**